spec.bs

```diff
@@ -732,7 +745,13 @@ following [=struct/items=]:
   :: a [=fencedframetype/fenced frame reporting map=]
 
   : <dfn>direct seller is seller</dfn>
```

Comment:
I don't find where this field is set. It seems it always uses the default `true`. Is that expected?

Reply:
It's set here:

https://wicg.github.io/turtledove/#:~:text=Set%20pendingConfig%E2%80%99s%20fenced%20frame%20reporting%20metadata%20to%20a%20struct%20with%20the%20following%20items%3A

Reply:
oh I see. Missed that, because clicking on the field cannot show usages like this. Should have done a search.
feel free to close this comment then.